app/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,47 @@
+// this has to be moved to container later, put it here because of conflicts
+.arts-container {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 100px;
+  margin-bottom: 20px;
+    a {
+      text-decoration: none;
+      color: white !important;
+    }
+}
+
+.art-card {
+  display: flex;
+  flex-direction: column;
+}
+
+.art-card-img {
+  box-sizing: border-box;
+  border-radius: 5%;
+  object-fit: cover;
+  height: 300px;
+  width: 300px;
+  margin-bottom: 5px;
+}
+
+.art-card-body {
+  margin-left: 5px;
+}
+
+.art-card-title {
+  font-weight: bold;
+}
+
+.art-card-text {
+  color: lightslategrey;
+}
```

Comment:
Can you try using $gold for title color and lightgrey for text color ?
It would fit better with overall design as they're already existing colors but maybe that's not the kind of design you want.

Another solution that is used in the forms is lightgrey for the title and lightgrey with an opacity of 0.5 for the text.

It doesn't really matter at this point so you can choose whichever you prefer

Reply:
changed this too